src/Adapter/Module/Tab/ModuleTabUnregister.php

```diff
@@ -72,7 +72,7 @@ public function __construct(TabRepository $tabRepository, LangRepository $langRe
      *
      * This is done automatically as part of the module uninstallation.
      *
-     * @return bool returns true if the module tabs were successfully uninstalled, false if any of them failed to do so
+     * @return void
```

Comment:
Can we check usages of `Module::unregisterTabs()`

I think the phpDoc is right and the code is wrong 😅  instead of the opposite

Reply:
:warning: it's not the scope of the PR ;)

Reply:
But the new PHPDoc is right ;)